### PR TITLE
Move frontmatter redirects to file

### PR DESCRIPTION
**Pull request by ProfessorLogout**
## Proposed change

This implements the changes from #14662

## Type of change
- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: #14662

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_redirects [hunk 1]
```diff
@@ -2066,6 +2067,11 @@
 /integrations/switch.knx /integrations/knx#switch
 /integrations/weather.knx /integrations/knx#weather
 /integrations/zigbee /integrations/xbee
+/integrations/hassio/ /hassio
+/integrations/cloud.alexa/ /cloud/alexa
+/getting-started/automation-create-first/ /getting-started/automation
+/getting-started/automation-2/ /getting-started/automation
```
Comment (klaasnicolaas) [inline source/_redirects]: @frenck Why do we use a `2` here? 🤔

Comment (frenck) [reply source/_redirects]: That is a pre-history URL, but still, it is referenced on some places.

File: source/_redirects [hunk 2]
```diff
@@ -76,6 +76,7 @@
 /developers/rest_api https://developers.home-assistant.io/docs/api/rest/
 /developers/server_sent_events https://developers.home-assistant.io/docs/external_api_server_sent_events/
 /developers/websocket_api https://developers.home-assistant.io/docs/api/websocket/
+/developers/code_of_conduct/ /code_of_conduct
```
Comment (frenck) [inline source/_redirects]: For the `_redirects` file, this last slash of the URL to redirect, can be removed. Both `/developers/code_of_conduct/` and `/developers/code_of_conduct` will work at that point.

```suggestion
/developers/code_of_conduct /code_of_conduct
```

This applies to all other redirects added as well.

File: source/_redirects (hunk 1 above)
Comment (klaasnicolaas) [reply source/_redirects]: Zero results on the docs website, and only 11 links on the community forum topic from 2017 and 2018 🤔

Comment (frenck) [reply source/_redirects]: Yep, and google as well. There is no need to ditch it if we can redirect people. So this is fine.

File: source/_redirects [hunk 3]
```diff
@@ -2066,6 +2067,11 @@
 /integrations/switch.knx /integrations/knx#switch
 /integrations/weather.knx /integrations/knx#weather
 /integrations/zigbee /integrations/xbee
+/integrations/hassio/ /hassio
+/integrations/cloud.alexa/ /cloud/alexa
+/getting-started/automation-create-first/ /getting-started/automation
```
Comment (frenck) [inline source/_redirects]:
```suggestion
/integrations/hassio /hassio
/integrations/cloud.alexa /cloud/alexa
/getting-started/automation-create-first /getting-started/automation
```

File: source/_redirects (hunk 1 above)
Comment (frenck) [inline source/_redirects]:
```suggestion
/getting-started/automation-2 /getting-started/automation
```

File: source/_redirects [hunk 4]
```diff
@@ -2135,6 +2141,7 @@
 /lovelace/views /lovelace/dashboards-and-views
 /lovelace/yaml-mode /lovelace/dashboards-and-views
 /lovelace/calendar-card /lovelace/calendar
+/integrations/lovelace/ /lovelace
```
Comment (frenck) [inline source/_redirects]:
```suggestion
/integrations/lovelace /lovelace
```

**Review by frenck — APPROVED** [review]
Made a couple of tweaks, and removed the WIP status, as this is the last bit to resolve the cleanup.

Thanks for the PR @ProfessorLogout 👍 

Happy Hacktoberfest 🎉

**Pull request merged**